Sources/CorePayments/Networking/GraphQL/GraphQLClient.swift

```diff
@@ -45,14 +54,10 @@ class GraphQLClient {
     }
 }
 
-extension GraphQLQuery {
+extension GraphQLQuery where Self: Codable {
 
-    func requestBody() throws -> Data {
-        let body: [String: Any] = [
-            "query": query,
-            "variables": variables
-        ]
-        let data = try JSONSerialization.data(withJSONObject: body, options: [])
-        return data
+    public func requestBody() throws -> Data {
```

Comment:
Should this include a `nodoc` since it's public but I assume just for internal use?

Reply:
I think that was because it is used in cardModule.

Reply:
5bd278901937af1b0ea2384f1adef4e884174d95